### PR TITLE
Logging for dashboard

### PR DESCRIPTION
**Pull request by raamSoftwire**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ app.post('/admin/company/new', auth, async (req, res) => {
 
 app.get('/company/:code', async (req, res) => {
   const { code } = req.params
-  const company = await getCompanyName(code)
+  const company = await getCompany(code)
```
Comment (jamiehumphries) [inline index.js]: ⚠️ You need to change the response to use `company.name` and handle `null`.

File: index.js [hunk 2]
```diff
@@ -98,24 +98,37 @@ io.on('connection', socket => {
   let company = null
   let userPseudonym = null
 
-  const formatMessage = message => `[${company || 'Unknown'} | ${userPseudonym || 'Anonymous'}] ${message}`
+  const formatMessage = message => `[${company.name || 'Unknown'} | ${userPseudonym || 'Anonymous'}] ${message}`
```
Comment (jamiehumphries) [inline index.js:101]: You need to properly handle `null` here for when a company is not identified.

File: index.js (hunk 1 above)
Comment (jamiehumphries) [inline index.js:77]: You need to change the response to use `company.name` and handle `null`.

File: index.js [hunk 3]
```diff
@@ -98,24 +98,37 @@ io.on('connection', socket => {
   let company = null
   let userPseudonym = null
 
-  const formatMessage = message => `[${company || 'Unknown'} | ${userPseudonym || 'Anonymous'}] ${message}`
+  const formatMessage = message => `[${company.name || 'Unknown'} | ${userPseudonym || 'Anonymous'}] ${message}`
   const info = message => console.log(formatMessage(message))
```
Comment (jamiehumphries) [inline index.js:102]: You can remove this `info` function now.

**Review by jamiehumphries — CHANGES_REQUESTED** [review]
The main thing here is handling the case where you don't get a company returned. It may be simplest to reintroduce a `getCompanyName(code)` function that does the null check for you, rather than doing it multiple times. Ask if you are not sure what I mean!

File: index.js (hunk 3 above)
Comment (raamSoftwire) [reply index.js:102]: I left it in as it is still used on lines 111 and 114.

**Pull request merged**